package.json

```diff
@@ -3,11 +3,12 @@
   "version": "0.0.2",
   "license": "MIT",
   "devDependencies": {
+    "core-js": "^3.1.3",
```

Comment:
Whats this?

Reply:
Storybook 5.1.1 updates it's Babel dependency without updating it's core-js dep. The result is a wall of core-js errors. They're aware of it and supposedly they're going to fix it, but this is the workaround in the meantime.

Reply:
Aaaaand, they just fixed it. I'm gonna get rid of this.